test/Syntax/Parser/diags.swift

```diff
@@ -0,0 +1,18 @@
+// REQUIRES: syntax_parser_lib
+// RUN: %swift-syntax-parser-test %s -dump-diags 2>&1 | %FileCheck %s
+
+// CHECK: [[@LINE+2]]:11 Error: consecutive statements on a line must be separated by ';'
+// CHECK: ([[@LINE+1]]:11,[[@LINE+1]]:11) Fixit: ";"
+let number⁚ Int
+// CHECK: [[@LINE-1]]:11 Error: operator with postfix spacing cannot start a subexpression
+
+// CHECK: [[@LINE+1]]:1 Error: expressions are not allowed at the top level
```

Comment:
Emitting this error is problematic, that means this error is emitted if you pass the source of a main swift file or a script-like one. Could you look into enabling whatever parser or language option will allow the parser to not complain about this ?

Reply:
Using `SourceFileKind::Main` will avoid this diagnostics.